xontrib/branch_expansion.py

```diff
@@ -32,7 +33,11 @@ def expand_braces(cmd, **_):
                 prefix + option + postfix for option in options.split(",")
             ]
             expanded_cmds.extend(expanded_parts)
+            requires_expansion = True
```

Comment:
Why not just return here? No need to wait

Reply:
the `{}` can exist in more parts